apps/package.json

```diff
@@ -112,9 +90,7 @@
     "eslint-plugin-react-hooks": "^4.6.0",
     "eslint-plugin-storybook": "^0.6.15",
     "events": "^3.3.0",
-    "fast-memoize": "2.0.2",
     "file-loader": "^6.2.0",
-    "firebase": "^2.4.2",
     "firebase-bolt": "^0.8.0",
     "firebase-mock": "^1.0.5",
     "firmata": "^2.2.0",
```

Comment:
I wonder if this could be moved to dependencies.

Reply:
Oh! Probably, I didn't know what `firmata` was, but I see you're the expert :-)

Reply:
done!